docs/source/top_level_elements/genre.rst

```diff
@@ -0,0 +1,163 @@
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+    <https://example.org/object/1> edm:hasType <http://id.loc.gov/authorities/genreForms/gf2014026131> .
+
+In 124 of the records in the VP Moore collection, with `@authority='lcgft'`, the `@authorityURI = 'http://id.loc.gov/authorities/genreForms'` is used; e.g. `vpmoore:50 <https://digital.lib.utk.edu/collections/islandora/object/vpmoore:50/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <genre authority="lcgft" authorityURI="http://id.loc.gov/authorities/genreForms" valueURI="http://id.loc.gov/authorities/genreForms/gf2014026173">Scrapbooks</genre>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+    <https://example.org/object/1> edm:hasType <http://id.loc.gov/authorities/genreForms/gf2014026173> .
+
+genre[@authority = 'lcmpt']
+---------------------------
+
+`@authority = 'lcmpt'` is used in the Van Vactor collection to express genre/performance medium instrumentation information; e.g. `vanvactor:12350 <https://digital.lib.utk.edu/collections/islandora/object/vanvactor:12350/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <genre authority="lcmpt" valueURI="http://id.loc.gov/authorities/performanceMediums/mp2013015074">bassoon</genre>
+    <genre authority="lcmpt" valueURI="http://id.loc.gov/authorities/performanceMediums/mp2013015342">horn</genre>
+    <genre authority="lcmpt" valueURI="http://id.loc.gov/authorities/performanceMediums/mp2013015748">trumpet</genre>
+    <genre authority="lcmpt" valueURI="http://id.loc.gov/authorities/performanceMediums/mp2013015540">percussion</genre>
+    <genre authority="lcgft" valueURI="http://id.loc.gov/authorities/genreForms/gf2014027156">Variations (Music)</genre>
+    <genre authority="lcgft" valueURI="http://id.loc.gov/authorities/genreForms/gf2014026956">Musical sketches</genre>
+    <genre authority="lcgft" valueURI="http://id.loc.gov/authorities/genreForms/gf2014026097">Excerpts</genre>
+    <genre authority="lcgft" valueURI="http://id.loc.gov/authorities/subjects/sh99001779">Scores</genre>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/object/1>
+        dcterms:subject <http://id.loc.gov/authorities/performanceMediums/mp2013015074> ;
+        dcterms:subject <http://id.loc.gov/authorities/performanceMediums/mp2013015342> ;
+        dcterms:subject <http://id.loc.gov/authorities/performanceMediums/mp2013015748> ;
+        dcterms:subject <http://id.loc.gov/authorities/performanceMediums/mp2013015540> ;
+        dcterms:type <http://id.loc.gov/authorities/genreForms/gf2014027156> ;
+        dcterms:type <http://id.loc.gov/authorities/genreForms/gf2014026956> ;
+        dcterms:type <http://id.loc.gov/authorities/genreForms/gf2014026097> ;
+        dcterms:type <http://id.loc.gov/authorities/subjects/sh99001779> .
+
+genre[@authority = 'lctgm']
```

Comment:
I removed this category of items by completing [https://jirautk.atlassian.net/browse/DIGITAL-833](https://jirautk.atlassian.net/browse/DIGITAL-833). This can be removed from the mapping.

Reply:
:+1:  I've updated the PR to reflect this.